recipes-connectivity/wifi/wifi-systemd-service.bb

```diff
@@ -0,0 +1,37 @@
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
+
+RDEPENDS_${PN} += " kernel-modules linux-firmware-rpidistro-bcm43455 wpa-supplicant "
```

Comment:
Works for me when I use `linux-firmware-rpidistro-bcm43430` on my RPi 3 Model B V1.2.

Reply:
Updated the recipe so it includes both version of drivers i.e. bcm43430 and bcm43455.